api/tinynews-models/src/plugins/graphql/resolver.ts

```diff
@@ -11,6 +11,37 @@ import { FieldResolver } from "@webiny/commodo-graphql/types";
 
 type GetModelType = (context: Object) => any; // TODO: add commodo type when available
 
+// export const resolveCreate = (getModel: GetModelType): FieldResolver => async (
```

Comment:
Will we end up needing this?

Reply:
Oh oops. I thought we would, but we don't as of right now. I'll remove it.